src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCollectorBatchBuilder.java

```diff
@@ -195,7 +200,8 @@ protected String buildItemKey(ConcurStandardAccountingExtractDetailLine saeLine)
         }
         
         return new StringBuilder(INITIAL_BUILDER_SIZE)
-                .append(makeEmptyIfBlank(saeLine.getChartOfAccountsCode()))
+                .append(makeEmptyIfBlank(saeLine.getReportId()))

```

Comment:
Yes, the Collector process is skipping line items that don't have a report ID. This method is mostly meant for creating a key to simplify consolidating similar transactions, and also for using that key for certain logging purposes.

I will simplify this to remove the "makeEmptyIfBlank" wrapper for the report ID, since the row should not even get to the consolidation point if it doesn't have a report ID.